components/event-bus/Gopkg.toml

```diff
@@ -37,26 +37,31 @@ required = [
   name = "github.com/stretchr/testify"
   version = "1.2.1"
 
-[[constraint]]
+[[override]]
```

Comment:
If we need to update or fix something in existing push, publish or sv, will it be a problem?

Reply:
No, I'm using it on my machine, but this upgrade is necessary now cause we are vendoring KN in Event-Bus. We will be more sure that all is working if Publish will be finished.